scripts/test-env.sh

```diff
@@ -92,5 +92,9 @@ kubectl kustomize "github.com/kubernetes-sigs/gateway-api/config/crd/experimenta
 
 echo "INFO: Updating helm dependencies"
 for i in charts/*; do
+  if [ "$i" = "charts/ingress" ]
+  then
+    continue
+  fi
```

Comment:
Can we have a more generic mechanism (for example: if `Chart.yaml` is not present) here?

Based on my previous comment: it's likely that the `ingress` chart will be reinstated in another PR very soon and we'll almost certainly forget to update this place.